core/src/test/java/org/infinispan/reactive/publisher/impl/SimpleClusterPublisherManagerTest.java

```diff
@@ -172,6 +173,39 @@ public void testCountSpecificKeys(DeliveryGuarantee deliveryGuarantee, boolean p
       assertEquals(expected, actualCount.intValue());
    }
 
+   @Test(dataProvider = "GuaranteeParallelEntry")
+   public void testCountEmptyKeys(DeliveryGuarantee deliveryGuarantee, boolean parallel, boolean isEntry) {
+      Cache<Integer, String> cache = cache(0);
+      int insertAmount = insert(cache).size();
```

Comment:
Nitpicking: Inserting 24 entries seems excessive in all the tests here, but even more when the test is not reading any of the entries.

Reply:
The test requires something to confirm it actually works properly. But I can just reduce all the tests from 24 to 12 for example maybe. TBH it isn't going to make that much of an actual difference. In prior examples we were inserting tens of thousands, which was obviously an issue.